packages/docs/pages/eng-onsite-2024.mdx

```diff
@@ -0,0 +1,44 @@
+import { Steps } from 'nextra/components'
+
+![Introduction](/assets/eng-challenge-2024.png)
+
+Hello, VTEX engineer! We’re thrilled to invite all engineers to join the Contribution Challenge.
+
+This is a chance to collaborate and make an impact on the Shoreline evolution. Whether you have experience with design systems or is a passionate beginner, your skills and ideas are invaluable to this initiative. Don't miss out on the opportunity to learn, grow, and leave your mark. Jump in, contribute, and let's code something amazing together!
+
+<Steps>
+
+### Introduction
+
+As the name suggests, this challenge is all about contributing to Shoreline, the VTEX design system for back-office experiences. Whether you're a frontend engineer or not, you're welcome to participate engage in any discussion, whether it's about engineering or design!
+
+This challenge demonstrates how an InnerSource process works in practice. We've provided some materials to guide you on how the process works, how to contribute, and how to kick off the challenge!
+
+1. [How to get started](https://shoreline.vtex.com/guides/contributing-introduction)
+2. [How the contribution process works](https://shoreline.vtex.com/guides/how-to-contribute)
+3. [Repository Organization](https://shoreline.vtex.com/guides/how-to-contribute#github)
+4. [Contributing to the Code](https://shoreline.vtex.com/guides/code/development-guideline)
+5. [Contributing to the Design](https://shoreline.vtex.com/guides/design/handoff-requirements)
+
+Feel free to open an issue, add a comment, or submit a pull request—and remember, this project belongs to all of us in the VTEX community!
+
+### Score
+
+To earn points, dive into contributing to Shoreline! As described [here](https://shoreline.vtex.com/guides/contributing-introduction#how-to-get-involved), there are plenty of ways to get involved and make an impact on the project! In this challenge, each contribution you make give you points:
+
+1. Join a discussion - 1 point per issue
+2. Open an issue - 2 points
+3. Submit a pull request - 2 points
+4. Get your pull request approved and merged - 3 points
+
+Every contribution counts, so jump in and start earning those points!
+
+### Winners Announcement
+
+The results will be announced on Wednesday (08/28) during the Shoreline presentation, and the top 3 participants will be rewarded! Don’t miss out on this opportunity to be part of this initiative!
```

Comment:
The results will be announced on Wednesday (Aug 28) during the Shoreline presentation and the top 3 participants will be rewarded. Don’t miss out on this opportunity to be part of this initiative!